systems/plants/RigidBodyManipulator.cpp

```diff
@@ -435,7 +435,7 @@ void RigidBodyManipulator::compile(void)
     // update the body's contact points
     Matrix3Xd contact_pts;
     getTerrainContactPoints(*bodies[i], contact_pts);
-    bodies[i]->contact_pts = contact_pts.colwise().homogeneous();
+    bodies[i]->contact_pts = contact_pts;
```

Comment:
why not `getTerrainContactPoints(*bodies[i], bodies[i]->contact_pts)` to save a copy?